scripts/gha/build_ios_tvos.py

```diff
@@ -0,0 +1,604 @@
+        platform_architecture_token = '{0}-{1}'.format(platform_variant,
+                                                       architecture)
+        archive_output_path = os.path.join(frameworks_os_path,
+                                                platform_architecture_token)
+        # Eg: <build_dir>/tvos_cmake_build/device-arm64
+        build_path = os.path.join(args.build_dir,
+                                  '{0}_cmake_build'.format(apple_os),
+                                  platform_architecture_token)
+        # For ios builds, we specify architecture to cmake configure.
+        architecture = architecture if apple_os == 'ios' else None
+        # For tvos builds, we pass a special cmake option PLATFORM to toolchain.
+        toolchain_platform = os_platform_variant_config['toolchain_platform'] \
+                             if apple_os == 'tvos' else None
+        # CMake configure was having all sorts of issues when run in parallel.
+        # It might be the Cocoapods that are downloaded in parallel into a
+        # single cache directory.
+        cmake_configure(args.source_dir, build_path,
+                        os_platform_variant_config['toolchain'],
+                        archive_output_path, architecture,
+                        toolchain_platform)
+        process = multiprocessing.Process(target=cmake_build,
+                                          args=(build_path, supported_targets))
+        processes.append((process, archive_output_path))
+
+  # Launch all cmake build processes in parallel.
+  for process, _ in processes:
+    process.start()
+
+  for process, archive_output_path in processes:
+    process.join()
+    # Reorganize frameworks (renaming, copying over headers etc)
+    arrange_frameworks(archive_output_path)
+
+  # if we built for all architectures build universal framework as well.
+  build_universal_framework(frameworks_path)
+
+  # Build xcframeworks
+  xcframeworks_path = os.path.join(args.build_dir, 'xcframeworks')
+  template_info_plist_path = os.path.join(args.source_dir, 'build_scripts',
+                                          'tvos', 'Info_ios_and_tvos.plist')
+  build_xcframeworks(frameworks_path, xcframeworks_path,
+                     template_info_plist_path)
+
+
+def parse_cmdline_args():
+  parser = argparse.ArgumentParser(description='Build for iOS and tvOS.')
+  parser.add_argument('-b', '--build_dir',
+    default='ios_tvos_build', help='Name of build directory.')
+  parser.add_argument('-s', '--source_dir',
+    default=os.getcwd(),
+    help='Directory containing source code (top level CMakeLists.txt)')
+  parser.add_argument('-v', '--platform_variant', nargs='+',
+    default=('device', 'simulator'),
+    help='List of platforms to build for.')
+  parser.add_argument('-a', '--architecture', nargs='+',
+    default=('arm64', 'armv7', 'x86_64', 'i386'),
+    help='List of architectures to build for.')
+  parser.add_argument('-t', '--target', nargs='+',
```

Comment:
added flag -t firebase_remote_config
Generate empty folders for other xcframeworks

Reply:
Fixed.